server/src/resolvers/ProgramingLanguageResolver.ts

```diff
@@ -0,0 +1,56 @@
+import { Args, Authorized, Mutation, Query, Resolver } from 'type-graphql';
+
+import { ProgramingLanguage } from 'models/ProgramingLanguage';
+import { ProgramingLanguageServices } from 'services/ProgramingLanguageServices';
+
+import { ErrorMessages } from 'utils/enums/ErrorMessages';
+import {
+	createLanguageArgs,
+	deleteLanguageArgs,
+	getLanguageByIdArgs,
+	updateLanguageArgs
+} from 'resolvers/args/ProgramingLanguageArgs';
+
+@Resolver(ProgramingLanguage)
+export class ProgramingLanguageResolver {
+	LanguageServices: ProgramingLanguageServices = new ProgramingLanguageServices();
```

Comment:
ProgrammingLanguageServices

(and update in other Resolvers)

Reply:
Done